src/Simplex/Messaging/Agent/Store/SQLite/Migrations/M_20210101_initial.hs

```diff
@@ -0,0 +1,142 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Messaging.Agent.Store.SQLite.Migrations.M_20210101_initial where
+
+import Database.SQLite.Simple.QQ (sql)
+import Simplex.Messaging.Agent.Store.SQLite.Migrations.Types (SchemaMigration (..))
+
+m_20210101_initial :: SchemaMigration
+m_20210101_initial =
+  SchemaMigration
+    { name = "20210101_initial",
+      up =
+        [sql|
+          CREATE TABLE IF NOT EXISTS servers(
```

Comment:
store these in from identically named .sql files? they're becoming hard to read w/t syntax highlighting

Reply:
moved to .sql files